src/components/BorrowerProfile/MoreAboutLoan.vue

```diff
@@ -1,6 +1,6 @@
 <template>
 	<section>
-		<div class="tw-prose">
+		<div class="tw-prose tw-overflow-auto">
```

Comment:
Seems ok. Also wonder if there is some "text-wrap" property that should be set here..

Reply:
`overflow-wrap: break-word;` could also work

Reply:
Nice!